build/configs/rtl8730e/rtl8730e_external.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+###########################################################################
+#
+# Copyright 2024 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# rtl8730e_download.sh
```

Comment:
this doesn't match the filename, also the naming seems not meaningful.
how about rtl8730e_ext_download.sh?